source/manual/alerts/data-sync-failed-to-munge.html.md

```diff
@@ -0,0 +1,29 @@
+---
+owner_slack: "#govuk-2ndline"
+title: URLs in data sync failed to munge
+section: Icinga alerts
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2018-06-25
+review_in: 3 months
+---
+
+As part of the data syncs, the router data should update to point to the relevant environment.
+
+If this fails, then on integration for example, apps will appear to be broken - you'll likely see `cache-1.router.integration.publishing.service.gov.uk;5xx rate for www-origin` alerts and 5xx's for frontend apps.
+
+Check URLs in router-api:
+
+```
+ssh router-backend-1.staging
```

Comment:
You might want to just generalise this to ssh onto a router machine since the command is different for integration and then different still for a draft machine

Reply:
👍